bluemira/utilities/tools.py

```diff
@@ -782,6 +782,78 @@ def polar_to_cartesian(
     return x, z
 
 
+def toroidal_to_cylindrical(R_0: float, z_0: float, tau: np.ndarray, sigma: np.ndarray):
+    """
+    Convert from toroidal coordinates to cylindrical coordinates in the poloidal plane
+    Toroidal coordinates are denoted by (\\tau, \\sigma, \\phi)
+    Cylindrical coordinates are denoted by (r, z, \\phi)
+    We are in the poloidal plane so take the angle \\phi = 0
+
+    .. math::
+        R = R_{0} \\frac{\\sinh{\\tau}}{\\cosh{\\tau} - \\cos{\\sigma}}
+        z = R_{0} \\frac{\\sin{\\tau}}{\\cosh{\\tau} - \\cos{\\sigma}} + z_{0}
+
+    Parameters
+    ----------
+    R_0:
+        r coordinate of focus in poloidal plane
+    z_0:
+        z coordinate of focus in poloidal plane
+    tau:
+        the tau coordinates to transform
+    sigma:
+        the sigma coordinates to transform
+
+    Returns
+    -------
+    [R, Z]:
+        Array containing transformed coordinates in cylindrical form
+    """
+    R = R_0 * np.sinh(tau) / (np.cosh(tau) - np.cos(sigma))  # noqa: N806
+    Z = R_0 * np.sin(sigma) / (np.cosh(tau) - np.cos(sigma)) + z_0  # noqa: N806
+    return [R, Z]
```

Comment:
I'd just return the tuple than bothering to create a list
```suggestion
    R, Z:
        Array containing transformed coordinates in cylindrical form
    """
    R = R_0 * np.sinh(tau) / (np.cosh(tau) - np.cos(sigma))  # noqa: N806
    Z = R_0 * np.sin(sigma) / (np.cosh(tau) - np.cos(sigma)) + z_0  # noqa: N806
    return R, Z
```